backend/backend/core/agents.py

```diff
@@ -28,12 +33,13 @@ def __init__(self, model, python_tool):
         super().__init__()
         prompt = ChatPromptTemplate.from_template(
```

Comment:
Consider adding a blank line between the template and the agent creation to improve readability.